### DISCUSSION TITLE
Adjusting ContextMenu font size in Avalonia

### DISCUSSION THREAD

**Discussion by ruxo** [AvaloniaCommunity/Material.Avalonia#369] [Ideas]
Hello everyone,

I'd like to customize the font size of my application's ContextMenu. The default size seems a bit large for my preference.

I've noticed that the MenuItem control template applies a local style to the AccessText element. This seems to override styles I define within the Window context. I'm fairly new to Avalonia, so I'd appreciate guidance on the following:

* Is there a recommended way to adjust the ContextMenu font size while working with these local styles?
* Would it be appropriate to propose moving the AccessText local style elsewhere in the MenuItem definition to allow greater customization?

![image](https://github.com/AvaloniaCommunity/Material.Avalonia/assets/639905/7ffec15b-c66e-49bc-9e8b-ac0c64989b60)

Thank you for your insights!

Best,
Rux

**Comment by SKProCH** [comment]
Hello, there is 2 ways:
- Copy the styles to your project, and paste it in you App.axaml after MaterialTheme (or in resources section, if you copy the ControlTheme)
- Create the Style with following selector: `AccessText#PART_MenuItemText" and set required values using setters.

As for `moving the AccessText local style elsewhere`, i've created an issue to track progress: #371

**Reply by ruxo** [reply]
Hello,

I've tried to set the style specifically as you suggested (targeting `AccessText#PART_MenuItemText`), but it doesn't work. The font size cannot override the local setting, as shown in the following screen capture.

![image](https://github.com/AvaloniaCommunity/Material.Avalonia/assets/639905/77ef1913-f601-4dfb-9d8e-e80b76bb077e)

I guess the only way is to override the control theme by copying the template. I'll try that later.

I appreciate your help. Thank you! :)

**Reply by SKProCH** [reply]
You can also use dev tools to look at the list of styles and final property values. There is probably something wrong with the selector.

**Reply by ruxo** [reply]
I did enable `attachDevTools` and tried to see the properties of the menu item's access text, via F12, and found that the font size is always 16, no matter which method I used to set the value 😔. I believe it always be overridden by that local style settings.

**Reply by SKProCH** [reply]
Which priority font size property has in devtools?
